src/main/java/de/cas/mse/exercise/diamond/Diamond.java

```diff
@@ -4,26 +4,26 @@
 
 public class Diamond {
 
-	public String print(int n) {
-		if (n <= 0 || n % 2 == 0) {
+	public String print(int size) {
+		if (size <= 0 || size % 2 == 0) {
 			return null;
 		}
-		StringBuilder builder = new StringBuilder(new String(make(n, n)));
-		for (int i = n - 2; i > 0; i -= 2) {
-			char[] chars = make(n, i);
+		StringBuilder builder = new StringBuilder(new String(createLayer(size, size)));
+		for (int i = size - 2; i > 0; i -= 2) {
+			char[] chars = createLayer(size, i);
 			builder.insert(0, chars);
 			builder.append(chars);
 		}
 		return builder.toString();
 	}
 
-	private char[] make(int i, int j) {
-		int amount = ((i - j) / 2);
-		char[] chars = new char[amount + j + 1];
+	private char[] createLayer(int height, int width) {
+		int amount = ((height - width) / 2);
+		char[] chars = new char[amount + width + 1];
```

Comment:
Line/Row would fit better. Or in this case. Layer since this method is called layer.